lotteryPowerBall.py

```diff
@@ -0,0 +1,52 @@
+import random
+AMOUNT_OF_MOST_OFTEN_NUM = 10
+
+
+# first 5 numbers need to be in range [1, 69]
+# the last one - is Power Ball - in range [1, 26]
+def create_lottery_numbers():
+    file = open('numbers.txt', 'w')
+    for i in range(100):
+        power_num = str(random.randint(1, 26)).zfill(2)
+        for j in range(5):
+            other_numbers = str(random.randint(1, 69)).zfill(2)
+            file.write(f'{other_numbers} ')
+        file.write(f'{power_num}\n')
+    file.close()
+
+
+#  create a dictionary with amount of each digits
+def set_of_digits_from_file():
+    file = open('numbers.txt', 'r')
+    list_of_digits = []
+    set_of_digit = {}
+    for line in file:
+        list_of_digits.extend(line.split())
+    file.close()
+    for digit in set(list_of_digits):
+        set_of_digit[digit] = list_of_digits.count(digit)
```

Comment:
`list_of_digits.count(digit)` needs to loop through entire list, so using it is expensive. There is a way to avoid using it entirely